pkgs/development/python-modules/aigyminsper/default.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  wheel,
+}:
+
+buildPythonPackage rec {
+  pname = "aigyminsper";
+  version = "0.2.3";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-QIT6qeSfLOLNhn5CFIlKaKn/gvZDwcvFxJFvBTCwQ4g=";
+  };
+
+  build-system = [
+    setuptools
+    wheel
+  ];
+
```

Comment:
We are not running the tests https://github.com/Insper/ai_gym/tree/main/tests , do we?